src/hooks/useRdfCollection.js

```diff
@@ -70,6 +70,7 @@ const useRdfCollection = (parse, serialize, fileUrl, fetchData) => {
   const addMutation = useMutation({
     mutationFn: async (item) => {
       if (!data) await fetchDocument();
+      // HERE
```

Comment:
Think we could remove this comment now that we found where to debug the problem.

Reply:
Yep my bad forgot I left it in honestly 😅